### ISSUE TITLE
#MAP function does not work with name 'Hsomething'

### ISSUE THREAD

**Issue by ryu-bu** [CIDARLAB/pyLFR#20]
Code: 
```
module transformation (
    finput B, C,
    foutput d_out
);

    // oil (carrier load) ignored
    // add plasmid and cell suspension
    flow mixture;
    assign mixture = B + C;

    // mixer is a heat shock mixer
    #MAP 'Hd' '~'
    assign d_out = ~mixture;

endmodule
```

Error:
```
line 12:9 extraneous input ''Hd' expecting '''
line 12:14 mismatched input ''' expecting ID
Stopping compiler because of syntax errors
```

This error occurs when the Map is named with any letters followed by 'h' or 'H'. 'h' or 'H' by themselves worked.

**Comment by rkrishnasanka** [comment]
@ryu-bu thanks for finding this! Looks like the grammar is detecting this as a HEX number, that's the reason why this is breaking. Its a tricky but easy fix.

**Comment by rkrishnasanka** [comment]
The issue is now fixed in commit - 5476dc7484a1a3d1ee94c2fba22b8f6b9446d1ea, however, since its a lexer level problem, I had to shift the syntax to use double quotes instead of single quotes instead.
